internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -864,9 +846,8 @@ func getPullRequestInfoFromPayload(
 	}, nil
 }
 
-func reconcilePrWithDb(
+func (s *Server) reconcilePrWithDb(
```

Comment:
Changed some functions to methods so we do not need to pass around the `db.Store` pointer.

Reply:
Long-term, we want these to be in the provider implementation and not the server, correct?

Reply:
Eventually, yes.